jest.config.js

```diff
@@ -23,7 +23,8 @@ const sharedConfig = {
     "testRegex": "__tests__.*\\.(spec|test)\\.ts$",
     "moduleFileExtensions": [
         "ts",
-        "js"
+        "js",
+        "json"
```

Comment:
Because otherwise it wasn't recognizing the package that is a dictionary for SPDX package versions to their full names. That package was just a JSON file, so the require in tests was failing.